### PR TITLE
chore: configure publishing packages

### PR DESCRIPTION
**Pull request by satya164**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,6 +16,7 @@ web-report/
 
 npm-debug.*
 yarn-error.log
+lerna-debug.log
```
Comment (thymikee) [inline .gitignore]: feel free to ignore all `*.log` files

File: README.md [hunk 2]
```diff
@@ -487,3 +487,13 @@ Then run the tests:
 ```sh
 yarn test
 ```
+
+## Publishing
+
+To publish a new version, first we need to to export a `GH_TOKEN` environment variable as mentioned [here](https://github.com/lerna/lerna/tree/master/commands/version#--create-release-type). Then run:
```
Comment (thymikee) [inline README.md]:
```suggestion
To publish a new version, first we need to export a `GH_TOKEN` environment variable as mentioned [here](https://github.com/lerna/lerna/tree/master/commands/version#--create-release-type). Then run:
```

**Review by thymikee — APPROVED** [review]
👍

**Review by osdnk — CHANGES_REQUESTED** [review]
Typo in title is unacceptable xD

**Pull request merged**